_data/tags.yml

```diff
@@ -12,3 +12,4 @@ allowed-labels:
   - release notes
   - tutorials
   - videos
+  - integrations list
```

Comment:
@vikramraman 
We want all the integrations documentation pages labeled with "integrations list" so that we can have a page like this: https://docs.wavefront.com/label_query%20language.html
@rkempf42 can confirm.